xbmc/addons/AddonInfo.h

```diff
@@ -76,9 +81,15 @@ namespace ADDON
     ADDON_MAX
   } TYPE;
 
+  typedef std::map<std::string, std::string> InfoMap;
+  typedef std::map<std::string, std::pair<const AddonVersion, bool> > ADDONDEPS;
```

Comment:
FYI, having two >> next to each other isn't a problem anymore since switching to c++11